cursor.go

```diff
@@ -110,7 +111,7 @@ func (c *Cursor) Prev() (key []byte, value []byte) {
 	return k, v
 }
 
-// Seek moves the cursor to a given key and returns it.
+// Seek moves the cursor to a given key using a binary search and returns it.
```

Comment:
nit: I would call it "b-tree search" (binary search is only within a single page)

Reply:
Done